dashboard/legacy/middleware/files_api.rb

```diff
@@ -428,8 +439,12 @@ def put_file(endpoint, encrypted_channel_id, filename, body)
       # TODO(JillianK): we are temporarily ignoring address share failures because our address detection is very broken.
       # Once we have a better geocoding solution in H1, we should start filtering for addresses again.
       # Additional context: https://codedotorg.atlassian.net/browse/STAR-1361
-      if share_failure && share_failure[:type] != "address"
-        details_key = share_failure.type == ShareFiltering::FailureType::PROFANITY ? "profaneWords" : "pIIWords"
+      if share_failure && share_failure[:type] != ShareFiltering::FailureType::ADDRESS
+        if share_failure[:type] == ShareFiltering::FailureType::PROFANITY
+          details_key = "profaneWords"
+        else
+          details_key = "pIIWords"
+        end
```

Comment:
Is this the same logic as before (with the ternary)? Any reason to change it (unless i'm missing something)?

Reply:
Oh yeah! I had changed this checking with `aichat`, but then updated to this logic comparing with my first change - https://github.com/code-dot-org/code-dot-org/pull/59414/commits/3ba5cb44226da04f29aa19f818af37fdacccd267#diff-7f7b9ffe903479398ea459bc7a01594b25b89baaa2c8fdda6aeb85deb7f9aacb.  So yes - same logic! Will revert.